docker/init_scripts/init

```diff
@@ -161,6 +161,21 @@ start_bin_valkey-server() {
 	error_log "Internal valkey did not become ready after $((max_retries * 500))ms"
 }
 
+# Commands to start RQ scheduler
+start_bin_scheduler() {
+	info_log "Starting RQ scheduler"
+
+	RQ_REDIS_HOST=${REDIS_HOST:-127.0.0.1} \
+		RQ_REDIS_PORT=${REDIS_PORT:-6379} \
+		RQ_REDIS_USERNAME=${REDIS_USERNAME:-""} \
+		RQ_REDIS_PASSWORD=${REDIS_PASSWORD:-""} \
+		RQ_REDIS_DB=${REDIS_DB:-0} \
+		RQ_REDIS_SSL=${REDIS_SSL:-0} \
```

Comment:
do these env vars work? the docs say you need `-H or --host: Redis server to connect to` https://github.com/rq/rq-scheduler